packages/components/src/components/icon/icon.tsx

```diff
@@ -89,6 +89,15 @@ export class Icon {
     );
 
     // TODO: wouldn't it be better to set alt attribute instead of aria-label?
-    return <img src={buildIconUrl(this.source || this.name)} {...parseAndGetAriaAttributes(this.aria)} alt="" />;
+    return (
+      <img
+        src={buildIconUrl(this.source || this.name)}
+        {...parseAndGetAriaAttributes(this.aria)}
+        width={24} // improve bootstrapping behaviour
+        height={24} // improve bootstrapping behaviour
+        loading="lazy"
```

Comment:
Wouldn't it be better if we'd make this configurable by a prop?

Reply:
As discussed in private chat, we always want to add the native prop as there is no disadvantage